starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,82 @@
 // Names and Input
 
+var hacker1 = "Laura";
+console.log("The driver's name is " + hacker1);
+
+var hacker2 = "Régis";
+console.log("The navigator's name is " + hacker2);
```

Comment:
You can interact with the user by asking his name dynamically with the prompt command : 
```
var hacker2 = prompt("What is the navigator's name?");
console.log("The navigator's name is " + hacker2);
```